examples/with-vuetify/nuxt.config.js

```diff
@@ -11,8 +11,7 @@ module.exports = {
   ],
   head: {
     link: [
-      { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Roboto' },
-      { rel: 'stylesheet', href: 'https://fonts.googleapis.com/icon?family=Material+Icons' }
```

Comment:
Had to manually include icons in [css/vendor/material-icons.styl](https://github.com/nuxt/nuxt.js/pull/703/files#diff-6da00ba1eef1fd4ab27becb008b85ffe), as preloading the material icons stylesheet does not load the font (for some reason I couldn´t figure out).